picard/script/serializer.py

```diff
@@ -166,6 +166,38 @@ def update_from_dict(self, settings):
         if updated and 'last_updated' not in settings:
             self.update_last_updated()
 
+    def to_dict(self):
+        """Generate a dictionary containing the object's OUTPUT_FIELDS.
+
+        Returns:
+            dict: Dictionary of the object's OUTPUT_FIELDS
+        """
+        items = {key: getattr(self, key) for key in self.OUTPUT_FIELDS}
+        items["script"] = str(items["script"])
```

Comment:
Why is this `str()` needed? Aren't they always strings?

Reply:
Actually the "script" value is returned as a `MultilineLiteral`.

Reply:
Ah ok :)